docs/code-snippets/outlook-snippets.yaml

```diff
@@ -195,6 +253,10 @@ Office.AppointmentCompose.saveAsync:
         "value":"AAMkADI5...AAA=",
         "status":"succeeded"
     }
+Office.AppointmentCompose.seriesId:
+  - |-
+    var seriesId = Office.context.mailbox.item.seriesId; 
+    var isSeries = (seriesId == null);
```

Comment:
Is there a common use-case for this null check? It might be in the ref docs already, but an explanation as to why you're doing this (and why it might be null) might be useful.